### PR TITLE
feat(losses): model_time on equilibrium matching and negate_velocity on flow matching

### LINKED ISSUE #283: EquilibriumMatchingLoss: option for the clock shown to the model (model_time)
## Problem

Since 0.8 the loss zeroes the clock unconditionally before every model call (`t_model = torch.zeros_like(t)` in `forward`, and `_probe_forward` does the same). 0.7.x exposed `time_invariant=False` to pass the sampled `t`; that escape is gone, so a time-conditioned field cannot be trained with this loss any more, and nothing fails: the model silently trains time-blind.

A concrete consequence is in our own docstring: `ct='constant'` with `ct_multiplier=1` is described as "exactly the negated Flow Matching objective". With a zeroed clock it is not, because FM's velocity field is time-conditioned; today that endpoint matches FM in the target only.

## Proposal

One constructor option, default preserving current behaviour:

```python
model_time: Literal["zero", "true"] | Callable[[torch.Tensor], torch.Tensor] = "zero"
```

- `"zero"`: the current EqM convention (field trained and sampled at t = 0).
- `"true"`: the sampled `t` is passed (restores what 0.7's `time_invariant=False` did).
- callable `t -> t'`: elementwise map on the `(batch,)` clock, for schedules and reparametrisations.

Applied at the training call and in `_probe_forward`, so the conditioning probe follows the same convention. Validate the literal in `__init__`, record it in `repr`.

## Sketch

```python
def _model_t(self, t):
    if callable(self.model_time):
        return self.model_time(t)
    return t if self.model_time == "true" else torch.zeros_like(t)
```
with `t_model = self._model_t(t)` in `forward` and `self.model(px, self._model_t(t0), **pmk)` in `_probe_forward`.

## Acceptance

- default path bit-identical to 0.8.5 (`t_model` all zeros);
- `model_time="true"`, `ct="constant"`, `ct_multiplier=1` equals `FlowMatchingLoss` on the same batch and `t` up to the target sign;
- a unit test showing a time-conditioned toy model's loss differs between `"zero"` and `"true"`, and does not for a time-invariant one.

## Resolution

Shipped as `time_invariant: bool = True` (the paper's term and the 0.7.x name, so `time_invariant=False` works again). True keeps the zeroed clock; False passes the sampled `t`. The callable clock map is not included; the conditioning probe always runs at t = 0. With `ct="constant"`, `ct_multiplier=1` and `time_invariant=False` the objective is bit-identical to `FlowMatchingLoss(negate_velocity=True)`.

### LINKED ISSUE #284: FlowMatchingLoss: negate_velocity option so an FM field can be sampled by the descent samplers
## Problem

`FlowMatchingLoss` regresses onto the positive velocity `x1 - x0` (paired with `FlowSampler`), while the EqM losses regress onto `-u_t * c(t)`, the convention the gradient-descent samplers and `EqMEnergy` assume (they subtract the field). A field trained with `FlowMatchingLoss` therefore cannot be dropped into the descent samplers without a user-side sign wrapper, and nothing warns.

## Proposal

`negate_velocity: bool = False` on `FlowMatchingLoss`, mirroring the sampler flag of the same name: when true, the target is `x0 - x1`. Document both conventions in one table shared by the FM and EqM docstrings.

## Acceptance

- `negate_velocity=False` bit-identical to today;
- with `negate_velocity=True`, the loss equals `EquilibriumMatchingLoss(ct="constant", ct_multiplier=1, model_time="true")` bit-for-bit on the same batch and `t` (depends on the `model_time` issue).

### PR DESCRIPTION
**Pull request by soran-ghaderi**
EquilibriumMatchingLoss gains model_time ('zero' default, 'true', or a callable t -> t') controlling the clock shown to the model at the training call and in the conditioning probe, restoring time-conditioned training. FlowMatchingLoss gains negate_velocity so a field trained with it drops into the descent samplers and EqMEnergy; with ct='constant', ct_multiplier=1 and model_time='true' the two objectives are bit-identical, which is pinned by tests. The field-sign and clock conventions of both losses are tabulated in the objectives concept page. Deprecation since stamps are corrected to the 0.8.5 release that shipped the warnings.

resolves #283
resolves #284

### CODE REVIEW

**Review by soran-ghaderi — COMMENTED** [review]
done
